### ISSUE TITLE
(Self-Voicing) Feasibility: Display content of read-me buttons on the screen somewhere

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#232]
I started this question on Slack, and transferring the question here:

@jessegreenberg, how hard or easy do you think it would be to make some a box of text pop up when a learner pressed the read me buttons? I am trying to get a sense if we would have time to explore the idea of providing visual access to the read me button content on the screen.

Displaying voiced text on the screen is the only thing I we haven't yet explored. Personally, the only place it seems useful in my mind for sims is with the read-me buttons.

A couple of participants have said it could be nice for some users to be able to re-read the content at their own pace. My inclination is that we may want to wait on developing this part of the self-voicing feature until after first publication, but before making that decision, I just wanted to see about feasibility.

@jessegreenberg responded:
...technically, I don't see this being difficult at all.

Just showing what is being spoken is arbitrary. I think the challenge will be designing what it looks like and making sure that it works in a way that doesn't occlude other sim components.

**Comment by terracoda** [comment]
Based on @jessegreenberg response, I will think about the visual design, and possibly include this idea in the design of the option menu and tool menus which are happening in https://github.com/phetsims/joist/issues/652

**Comment by terracoda** [comment]
I feel that "surfacing the text of the Read-Me" button content will be valuable, but it will have to be part of future design work. Putting this issue on hold for now.

**Comment by jessegreenberg** [comment]
This is not a sim-specific consideration, would you like to move this to a design document or a11y-research @terracoda?
